Presentation.Web/app/components/it-contract/it-contract-overview.controller.ts

```diff
@@ -745,6 +745,20 @@
                             dataItem && dataItem.status && `Hvid: ${dataItem.status.white}, Rød: ${dataItem.status.red}, Gul: ${dataItem.status.yellow}, Grøn: ${dataItem.status.green}, Max: ${dataItem.status.max}` || "",
                         sortable: false,
                         filterable: false
+                    },
+                    {
+                        field: "RepurchaseInitiated", title: "Genanskaffelse igangsat", width: 150,
```

Comment:
It's in the wrong overview.. place it before the other "Genanskaffelse**" columns in the other one